simulations/vip-483/bscmainnet.ts

```diff
@@ -0,0 +1,222 @@
+import { mine, time } from "@nomicfoundation/hardhat-network-helpers";
+import { expect } from "chai";
+import { ethers } from "hardhat";
+import { NETWORK_ADDRESSES } from "src/networkAddresses";
+import { forking, testVip } from "src/vip-framework";
+
+import vip483 from "../../vips/vip-483/bscmainnet";
+import CORE_POOL_RATE_MODEL_ABI from "./abi/JumpRateModel.json";
+import RATE_MODEL_ABI from "./abi/JumpRateModelV2.json";
+import POOL_REGISTRY_ABI from "./abi/PoolRegistry.json";
+import {
+  RateCurvePoints,
+  VTokenContractAndSymbol,
+  getAllVTokens,
+  getCorePoolRateCurve,
+  getPoolVTokens,
+  getRateCurve,
+} from "./common";
+
+const BSCMAINNET_CHECKPOINT = 1745903100;
+
+const CORE_COMPTROLLER = "0xfD36E2c2a6789Db23113685031d7F16329158384";
+
+forking(48555946, async () => {
```

Comment:
I think this block number should be updated to something after the block where [VIP 484](https://app.venus.io/#/governance/proposal/484?chainId=56) has been executed, where a new IR for vBNB was set

```suggestion
forking(48613237, async () => {
```